daml-assistant/daml-helper/src/DamlHelper.hs

```diff
@@ -144,9 +144,16 @@ runSandbox port args = do
     exitCode <- withSandbox port args waitForProcess
     exitWith exitCode
 
-runStart :: FilePath -> IO ()
-runStart darPath = do
-    withSandbox sandboxPort [darPath] $ \sandboxPh -> do
+runStart :: IO ()
+runStart = withProjectRoot $ \_ -> do
+    projectConfig <- getProjectConfig
+    projectName :: String <-
+        requiredE "Project must have a name" $
+        queryProjectConfigRequired ["name"] projectConfig
+    let darName = projectName <> ".dar"
+    assistant <- getDamlAssistant
+    callProcess assistant ["build", "-o", darName]
```

Comment:
Will this always build? Should we add short circuiting to `daml build`?

Reply:
For now it will always build. We should definitely make sure that `daml build` does not always rebuild.

Reply:
Sure. Can we get a ticket to that effect?

Reply:
https://github.com/digital-asset/daml/issues/550